pkg/shell/index.html

```diff
@@ -126,234 +103,193 @@ <h4 class="modal-title" translate="yes">About Web Console</h4>
-                                                            value="" type="password"/>
-                                                    </dd>
-                                                    <dt>
-                                                        <label for="credential-new-passwd" translate="yes">New password</label>
-                                                    </dt>
-                                                    <dd>
-                                                        <input class="form-control credential-new"
-                                                            autocomplete="new-password"
-                                                            id="credential-new-passwd"
-                                                            value="" type="password"/>
-                                                    </dd>
-                                                    <dt>
-                                                        <label for="new-passwd-confirm" translate="yes">Confirm</label>
-                                                    </dt>
-                                                    <dd>
-                                                        <input class="form-control credential-two"
-                                                            autocomplete="new-password"
-                                                            id="new-passwd-confirm"
-                                                            value="" type="password"/>
-                                                    </dd>
-                                                    <dt>
-                                                        <a tabindex="0" role="button" data-toggle="popover"
-                                                            data-trigger="focus" data-placement="top"
-                                                            data-content="Tip: Make your key password match your login password to automatically authenticate against other systems." translate="data-content">
-                                                            <span class="fa fa-lg fa-info-circle"></span>
-                                                        </a>
-                                                    </dt>
-                                                    <dd>
-                                                        <button class="credential-change pf-c-button pf-m-primary"
-                                                            translate="yes">Change password</button>
-                                                    </dd>
-                                                </dl>
-                                            </div>
-                                        </td>
-                                    </tr>
-                                </tbody>
-                            </table>
+        <div hidden aria-hidden="true" id="credentials-dialog">
```

Comment:
This modal has the same issue as described here https://github.com/cockpit-project/cockpit/issues/14895

In this case tabbing works immediately, but is escapes the modals after the action buttons.
I think we can fix it in another PR, since this issue is now in many places.

Reply:
Yeah, I think lets do it in separate PR where we fix it in all places at once.